packages/docs/src/components/sidebar/Sidebar.vue

```diff
@@ -42,15 +42,15 @@
           >
             <va-sidebar-item-content>
               <va-sidebar-item-title>
-                {{ $t(childRoute.displayName) }}
+                {{ $tie(childRoute.displayName) }}
               </va-sidebar-item-title>
               <div class="va-sidebar-item-badges" v-if="childRoute.meta && childRoute.meta.badge">
                 <va-chip
                   size="small"
                   :color="badgeColors[childRoute.meta.badge]"
-                  :title="$t(`menu.badges.${childRoute.meta.badge}.title`)"
+                  :title="$tie(`menu.badges.${childRoute.meta.badge}.title`)"
```

Comment:
pretty sure that could only ever be translation.

Reply:
I returned it as it was